search/create_datastore_and_search.ipynb

```diff
@@ -0,0 +1,3120 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.        client = discoveryengine_v1alpha.EngineServiceClient()
</pre><p>You might need to add the client Options here as well.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/271/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>